woocommerce/woocommerce-ios
# [Mobile Payments] Update Info.plist. Enable location services, required background modes and bluetooth permissions

Closes #3593 

⚠️  This PR is against the Mobile Payments feature branch, not develop ⚠️ 

## Changes
Update the Info.plist file with the keys required by the Stripe Terminal SDK

## How to test
* Check that the diff in the Info.plist file corresponds with [what Stripe requires](https://stripe.com/docs/terminal/sdk/ios#configure).
* Check that the build passes CI.

Update release notes:
